ofrak_patch_maker/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ### Fixed
 - X86_64 toolchain now installs on Docker image builds for AARCH64 hosts. ([#405](https://github.com/redballoonsecurity/ofrak/pull/405))
 - Toolchain now drops the .altinstrs_replacement as well as the .altinstructions section in our generated linker scripts ([#414](https://github.com/redballoonsecurity/ofrak/pull/414))
+- Update patchmaker dockerstub to use mirror server for downloading the BCC toolchain ([#541](https://github.com/redballoonsecurity/ofrak/pull/541))
```

Comment:
This detail doesn't really matter to our users, since we have not released this.

For the 502 entry for toolchains, can you just list this PR as well? (502, 541). That gives users all they need to know at a higher level.